2021/1/13 11:41:22
用户未登入,请先登入
MealTicket_Admin_Handler.Model.WebApiException: 用户未登入,请先登入
   在 MealTicket_Admin_Handler.AuthorityHandler.CheckAccountLogin(String userToken, HeadBase baseData) 位置 F:\项目\饭票后端\MealTicket_Admin_Handler\AuthorityHandler.cs:行号 67
   在 MealTicket_Admin_APIService.Filter.CheckUserLoginFilter.OnActionExecuting(HttpActionContext actionContext) 位置 F:\项目\饭票后端\MealTicket_Admin_APIService\Filter\CheckUserLoginFilter.cs:行号 31
   在 System.Web.Http.Filters.ActionFilterAttribute.OnActionExecutingAsync(HttpActionContext actionContext, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__6.MoveNext()

2021/1/13 11:42:47
用户未登入,请先登入
MealTicket_Admin_Handler.Model.WebApiException: 用户未登入,请先登入
   在 MealTicket_Admin_Handler.AuthorityHandler.CheckAccountLogin(String userToken, HeadBase baseData) 位置 F:\项目\饭票后端\MealTicket_Admin_Handler\AuthorityHandler.cs:行号 67
   在 MealTicket_Admin_APIService.Filter.CheckUserLoginFilter.OnActionExecuting(HttpActionContext actionContext) 位置 F:\项目\饭票后端\MealTicket_Admin_APIService\Filter\CheckUserLoginFilter.cs:行号 31
   在 System.Web.Http.Filters.ActionFilterAttribute.OnActionExecutingAsync(HttpActionContext actionContext, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<ExecuteActionFilterAsyncCore>d__5.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.ActionFilterAttribute.<CallOnActionExecutedAsync>d__6.MoveNext()

2021/1/13 11:43:51
服务器内部错误
System.ComponentModel.Win32Exception (0x80004005): 等待的操作过时。

